tests/distributed/model_parallel_test_template.txt

```diff
@@ -0,0 +1,85 @@
+# This is a template file for testing model parallelization.
```

Comment:
Why is it a txt file?

Reply:
Because it is a template file, that is filled by tests to create a real python file that will be executed.